CHANGELOG.rst

```diff
@@ -2,6 +2,18 @@
 Changelog
 =========
 
+2.8.4 (2025-01-29)
+==================
+
+Regular bugfix release
+
+* [:backend:`5035`] Removed non-existing data shown in components variables and
+  legacy Objects API submission data.
+* Fixed broken links in redis documentation.
+* Pinned Github actions runners to Ubuntu version 22.04.
+* Upgraded the Django framework to version 4.2.18.
+* Updated eHerkenning documentation.
```

Comment:
Bugfix releases should basically have "Fixed ..." for *every* changelog entry. Additional CI fixes etc. can be omitted, it's not useful information to people that want to know which bugs are fixed (because they want to know if _their_ problems have been solved).

```suggestion
* [:backend:`5035`] Fixed the legay Objects API registration including fields from other steps in the submitted data.
* Applied the latest Django security patches.
```